src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/infrastructure/clients/RefDataApi.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.iacasepaymentsapi.infrastructure.clients;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.fee.OrganisationResponse;
+
+@FeignClient(name = "rd-professional-api", url = "${rd-professional.api.url}")
+public interface RefDataApi {
+
+    @GetMapping("/refdata/external/v1/organisations/pbas")
+    OrganisationResponse findOrganisation(
+        @RequestParam(name = "email") String email
+    );
```

Comment:
you need to add authentication headers like we have in `PaymentApi` headers

Reply:
Added